### DISCUSSION TITLE
erro ao vazer Building

### DISCUSSION THREAD

**Discussion by alexandr30novaes** [vercel/community#1535] [Help]
### Summary

Fui fazer a Building do meu projeto, e está dando erro, o erro é :<html>
<body>

sh: preact: command not found
--
20:02:54.974 | Error: Command "preact build" exited with 127

</body>
</html>

**Comment by amyegan** [comment]
Thanks for reaching out, Alexandre. Can you share your project's `package.json`? If you have a `vercel.json` config, that would also be helpful to see.
